CHANGELOG.md

```diff
@@ -22,6 +22,15 @@
 - Implement `DataStore::get_note_script()` for `NtxDataStore` (#[1332](https://github.com/0xMiden/miden-node/pull/1332)).
 - [BREAKING] Added `AccountTreeWithHistory` and integrate historical queries into `GetAccountProof` ([#1292](https://github.com/0xMiden/miden-node/pull/1292)).
 
+## v0.11.4 (TBD)
```

Comment:
Can we add a date here?

Reply:
Wasn't 0.11.4 deployed a couple of days ago?

Reply:
That was 0.11.3 https://github.com/0xMiden/miden-node/releases

I think there were some dangling tags that didn't get deleted when we undid a release or something? So we re-used 0.11.3 for that one. Changelog updated